README.md

```diff
@@ -97,6 +97,99 @@ In cases where this default variable wrapping can cause problems (e.g. a variabl
 
   `{{#wrap}}[{{nowrap link.text}}]({{nowrap link.url}}){{/wrap}}`
 
+#### Handlebars helpers
+
+In addition to the standard helpers which are part of Handlebars, there are a number of additional helpers defined to assist with Ricardian contract specific issues.
+
+___Since 0.0___
+* `wrap` - Takes no parameters. Block level helper to wrap the contents with `<div>` tags for highlighting. See `Variables` section above for more details.
```

Comment:
for the `Variables` mentions here and below - might be helpful to link the header in the README, like so [`Variables`](https://github.com/EOSIO/ricardian-spec/blob/19ea6c0224107fac5f721b6fc977bb43ad207b61/README.md#variables)